--- Makefile.old	2010-05-14 21:41:56.587547126 +0200
+++ Makefile	2010-05-14 21:41:42.173117894 +0200
@@ -31,14 +31,7 @@
 BINDIR=$(DESTDIR)/usr/bin
 LIBDIR=$(DESTDIR)/usr/lib/bup
 install: all
-	$(INSTALL) -d $(MANDIR)/man1 $(DOCDIR) $(BINDIR) \
-		$(LIBDIR)/bup $(LIBDIR)/cmd
-	$(INSTALL) -o 0 -g 0 -m 0644 \
-		$(wildcard Documentation/*.1) \
-		$(MANDIR)/man1
-	$(INSTALL) -o 0 -g 0 -m 0644 \
-		$(wildcard Documentation/*.html) \
-		$(DOCDIR)
+	$(INSTALL) -d $(BINDIR) $(LIBDIR)/bup $(LIBDIR)/cmd
 	$(INSTALL) -o 0 -g 0 -m 0755 bup $(BINDIR)
 	$(INSTALL) -o 0 -g 0 -m 0755 \
 		$(wildcard cmd/bup-*) \
