--- src/pacpl-4.0.4/plugins/dolphin/pacpl.desktop	2008-01-04 06:19:58.000000000 +0100
+++ plugins/dolphin/pacpl.desktop	2009-02-15 02:51:24.000000000 +0100
@@ -1,5 +1,7 @@
 [Desktop Entry]
-ServiceTypes=audio/*,video/*,application/x-flash-video,inode/directory
+Type=Service
+ServiceTypes=KonqPopupMenu/Plugin
+MimeType=audio/*;video/*;application/x-flash-video;inode/directory;
 Actions=AAC;AC3;AIF;AIFF;APE;AU;AVR;BONK;CAF;CDR;FAP;FLA;FLAC;IRCAM;LA;LPAC;M4A;MAT;MAT4;MAT5;MMF;MP2;MP3;MP4;MPC;MPP;NIST;OFR;OFS;OGG;PAC;PAF;PVF;RA;RAW;SD2;SF;SHN;SMP;SND;SPX;TTA;VOC;W64;WAV;WMA;WV;
 X-KDE-Submenu=PACPL-Convert
 
