diff -ru gx_mail_notify-0.4.3.orig//list_account_item.cpp gx_mail_notify-0.4.3/list_account_item.cpp
--- gx_mail_notify-0.4.3.orig//list_account_item.cpp	2009-02-18 18:47:55.000000000 +0100
+++ gx_mail_notify-0.4.3/list_account_item.cpp	2010-05-06 00:39:59.725778159 +0200
@@ -26,9 +26,9 @@
 	font_normal = this->font();
 	font_color_normal = this->foreground();
 
-	font_not_ready = QFont::QFont(font_normal);
+	font_not_ready = QFont(font_normal);
 	font_not_ready.setItalic(true);
-	font_color_not_ready = QBrush::QBrush(font_color_normal);
+	font_color_not_ready = QBrush(font_color_normal);
 	font_color_not_ready.setColor(Qt::gray);
 
 	updateStatus();
